wasmplugin/plugin.go

```diff
@@ -365,3 +394,51 @@ func logError(error ctypes.MatchedRule) {
 		proxywasm.LogDebug(msg)
 	}
 }
+
+type AddressInfo struct {
```

Comment:
Fine to just return tuple of two values from the functions

Reply:
Done